src/components/card/LetterCard.jsx

```
@@ -0,0 +1,54 @@
+import React from "react";
```

ESLint

[error] 'React' is defined but never used.

(no-unused-vars)